actors.md

```diff
@@ -18,50 +35,58 @@ Some state machine actors are 'system' actors that get instantiated in the genes
 | 0    | InitActor          | Network Init            |
 | 1    | AccountActor       | Network Treasury        |
 | 2    | StorageMarketActor | Filecoin Storage Market |
-|  99 | AccountActor | Burnt Funds |
+| 99   | AccountActor       | Burnt Funds             |
+
+### Init Actor
 
-## Init Actor
+- **Code Cid**: `<codec:raw><mhType:identity><"init">`
 
 The init actor is responsible for creating new actors on the filecoin network. This is a built-in actor and cannot be replicated. In the future, this actor will be responsible for loading new code into the system (for user programmable actors). ID allocation for user instantiated actors starts at 100. This means that `NextID` will initially be set to 100.
 
-```go
-type InitActor struct {
-	// Mapping from Address to ID, for lookups.
-	AddressMap map[Address]BigInt
+```sh
+type InitActorState struct {
+    addressMap {Address:ID}
```

Comment:
this will need to be a HAMT, I think we write this as:
`addressMap &{Address:ID}<HAMT>`

cc @warpfork

Reply:
Is there a way to say "all maps use hamts"? I would really like all of them in the actors to use hamts without having to remember this..

Reply:
fixed